backend/docs/db/schema.sql

```diff
@@ -234,7 +257,10 @@ CREATE VIEW public.external_metadata_view AS
             sequence_entries_preprocessed_data.finished_processing_at
            FROM public.sequence_entries_preprocessed_data
           WHERE (sequence_entries_preprocessed_data.pipeline_version = ( SELECT current_processing_pipeline.version
-                   FROM public.current_processing_pipeline))) cpd
+                   FROM public.current_processing_pipeline
```

Comment:
This is quite hard to read and probably clearer to read (and possibly also more performant) by joining the organism and current_processing_pipeline_version for a column.